otter/test/convergence/test_effecting.py

```diff
@@ -112,3 +114,36 @@ def test_multiple_requests_of_different_type(self):
                              headers=None,
                              data=data_sentinel)]
         self.assertCompileTo(conv_requests, expected_effects)
+
+
+@implementer(IStep)
+class Steppy(object):
+    """A dummy step."""
+    def as_request(self):
+        """Return a simple GET .../whatever on CLOUD_LOAD_BALANCERS"""
+        return Request(service=ServiceType.CLOUD_LOAD_BALANCERS,
+                       method="GET",
+                       path="whatever")
+
+
+class StepsToEffectTests(SynchronousTestCase):
```

Comment:
I agree; `_reqs_to_effect` is an implementation detail, and one that we're hoping to eliminate. However, this test should keep working.